src/ERC1155A.sol

```diff
@@ -280,9 +280,10 @@ abstract contract ERC1155A is IERC1155A, IERC1155Errors {
 
     /// @inheritdoc IERC1155A
     function registerSERC20(uint256 id) external virtual override returns (address) {
+        if (_totalSupply[id] == 0) revert ID_NOT_MINTED_YET();
```

Comment:
not present anymore in SuperPositions.sol
https://github.com/superform-xyz/superform-core/pull/313#discussion_r1375187848